metricbeat/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.10.3
```

Comment:
@ruflin Do you know why the version of Golang differed from other Beats versions? Is it ok to update it?
I haven't seen any related failures yet.

Reply:
This is odd, all Beats should be in sync :-( I assume this had only an affect on testing and not packaging, meaning in 7.0 all Beats are packaged with the same Golang version.

Please update, I assume we missed it in the last Golang update.

Reply:
It was downgraded here: https://github.com/elastic/beats/pull/9242/files#diff-69f23758930df5eda4eeb23437b8a2ccR1

It seems like an accidental change. Or at least I haven't seen any reasoning why it was done.

Reply:
@kvch Thanks for investigation. @sayden I left a comment on the old PR, I wonder if any of these changes should have happened there.

Reply:
It probably was a merge gone bad.

Reply:
@sayden Can you please open a PR with reverting to the old Dockerfile?